config.json

```diff
@@ -2218,6 +2218,23 @@
         ],
         "difficulty": 5
       },
+      {
+        "slug": "state-of-tic-tac-toe",
+        "name": "State of Tic Tac Toe",
+        "uuid": "905ac84d-7ba1-4208-bee1-55d8efde0c01",
+        "prerequisites": [
+          "strings",
+          "tuples",
+          "atoms",
+          "cond",
+          "lists",
+          "enum",
+          "maps",
+          "list-comprehensions"
+        ],
+        "practices": [],
+        "difficulty": 5
```

Comment:
What? You're saying this is easier than killer sudoku helper? 😱  It took me 4x longer to solve this than killer sudoku helper... I guess this can be a 5 or 6 if we bring killer sudoku helper down to 4, or maybe killer sudoku helper should be a 3 instead? I'm not sure... I haven't done any practice exercises in a long time.

Reply:
Haha, it didn't take me that long, I guess our initial approaches were different. I started out with the error cases and everything was smooth from there. 

Also, every time the input is a "2D" grid string, I got in the habit of turning into an "indexed" map of `{{row, col}, cell}` which really helps manipulation (1 line frequency count and 3 line `win?/2`). This pattern was born from solving so many practice exercises.